yaksh/templates/yaksh/preview_questionpaper.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
+
+{% block pagetitle %} Quiz: {{ paper.quiz.description }} {% endblock pagetitle %}
+
+{% block content %}
+<div class="well">
+	<div class="col-md-12">
+		<div class="col-md-6">Maximum Mark(s): {{ paper.total_marks }}</div>
+		<div class="col-md-6"><span class="pull-right">Total Time: {{ paper.quiz.duration }}</span></div>
```

Comment:
Please show time unit (minutes or hours).

Reply:
Done. Added to the change-set in the reopened PR